app/renderer/actions/Inspector.js

```diff
@@ -69,6 +70,9 @@ export const SET_ACTION_ARG = 'SET_ACTION_ARG';
 
 export const SET_CONTEXT = 'SET_CONTEXT';
 
+export const SET_APP_ID = 'SET_APP_ID';
+export const GET_SERVER_STATUS = 'GET_SERVER_STATUS';
```

Comment:
i still think this should be `SET_SERVER_STATUS`, since we are setting the server status state variable.